src/scripts/bad_medgen_xrefs_grep_command.py

```diff
@@ -0,0 +1,40 @@
+"""From a list of bad MedGen xrefs, generate a grep command to remove them from Mondo.
+
+See also:
+- GH issue: https://github.com/monarch-initiative/mondo-ingest/issues/273
+"""
+from argparse import ArgumentParser
+
+
+def run(input_file: str, output_file: str, target: str):
+    """Create grep command"""
+    bad_xref_cuis = []
+    with open(input_file, "r") as file:
+        for line in file:
+            bad_xref_cuis.append(line.strip())
+
+    output_file2 = output_file.replace('.txt', '-grep-command.sh')
+    command_string = "grep -v "
+    for s in bad_xref_cuis:
+        command_string += f"-e {s} "
+    command_string += f"{target} > {target + '.tmp'}\n"
```

Comment:
Also, this PR here should _only_ be about UMLS.

Reply:
1. UMLS
Yes, I was thinking about prepending "UMLS" to these search strings. Probably better to do that. But just FYI I checked the results, and only UMLS:<ID> cases were affected; no other namespaces.
2. `xref: {s}`
Yes, I was thinking about that. That is a good solution to some of [the `grep -v` problems](https://github.com/monarch-initiative/mondo/pull/6560#discussion_r1295270141). I'm about to read [your reply](https://github.com/monarch-initiative/mondo/pull/6560#discussion_r1295695067) to that. If we choose a different route than `grep -v`, then this point is moot. But if we stay with `grep -v`, I'll make this change.
3. Replace CUIs instead of delete
Maybe. But still doesn't address many [problems](https://github.com/monarch-initiative/mondo/pull/6560#discussion_r1295270141). I think what we decide there will decide how we do this.
<details><summary>Some more, maybe redundant thoughts</summary>
<p>

Not all CUIs need to be replaced; some just need to be removed. So we could do replacements where needed (that entails something like SPARQL UPDATE) and deletions where needed. But for the removals, sometimes they're being removed in too many places.
Replacing them here is also in conflict with 2 other options: (a) I just wanted to remove all of these xrefs and bring in new ones from the ingest anyway, (b) assuming not 'b', part '2' of [the OP](https://github.com/monarch-initiative/mondo/pull/6560#issue-1850743276), is about your instruction to provide a robot template to introduce these additions as a 2nd step, which will have same effect as doing the find/replace of CUIs using something like SPARQL UPDATE.

</p>
</details>